### ISSUE TITLE
installdep and Makefile on debian are not working

### ISSUE THREAD

**Issue by Nhoya** [visualboyadvance-m/visualboyadvance-m#64]
On debian the installscript is not working, this is caused by `libpng16-deb` not in the repositories. Another problem is that the version of Cmake on repositories is `3.0.2` and the required version is `3.3.2`

**Comment by rkitover** [comment]
Will fix this over the next few days, in the meantime you can lower the cmake version at the top of the root CMakeLists.txt and the package name in the debian_installdeps() function in `./installdeps` .

**Comment by Nhoya** [comment]
Forcing using 3.0.2 version will raise errors 
```
116 CMake Error at CMakeLists.txt:310 (ADD_COMPILE_OPTIONS):
 117   Error evaluating generator expression:
 118 
 119     $<COMPILE_LANGUAGE:C>
 120 
 121   Expression did not evaluate to a known generator expression
 122 
 123 
 124 CMake Error at CMakeLists.txt:314 (ADD_COMPILE_OPTIONS):
 125   Error evaluating generator expression:
 126 
 127     $<COMPILE_LANGUAGE:CXX>
```

**Comment by rkitover** [comment]
Fixed in 34e408cc for Debian 8 "Jessie" and Ubuntu 14 "Trusty".

**Comment by rkitover** [comment]
Ok, I think we have this all ironed out now. Closing this.
